src/translation/translation_es_ES.ts

```diff
@@ -136,7 +136,7 @@
     <message>
         <location filename="../util.cpp" line="730"/>
         <source>Korean</source>
-        <translation type="unfinished"></translation>
+        <translation>Coreano</translation>
```

Comment:
Did we discuss anywhere that we really translate all languages?

Reply:
This is in the 'About' window listing the languages and their translators. I think it makes sense to translate it.

Reply:
Ok.